mscore/workspace.cpp

```diff
@@ -169,10 +184,7 @@ void MuseScore::deleteWorkspace()
       if (reply != QMessageBox::Yes)
             return;
 
-      WorkspacesManager::remove(workspace->name());
-      QFile f(workspace->path());
-      f.remove();
-      delete workspace;
+      WorkspacesManager::remove(workspace);
```

Comment:
Is this safe for the Based_edited and Advanced_edited workspaces?  As mentioned on Telegram, current behavior is a crash.

Reply:
Tested, seems to work OK now.